### DISCUSSION TITLE
How should I integrate Blockscout into my local private chain?

### DISCUSSION THREAD

**Discussion by kyonRay** [blockscout/blockscout#10666] [Q&A] [answered]
Hi~
I currently have a locally deployed EVM-compatible blockchain (a blockchain I developed myself). How should I use BlockScout to connect to this blockchain? How can I determine which necessary JSON-RPC api I am missing? Is there any corresponding documentation available?

Very thank you for any clues.

**Comment by sl1depengwyn** [comment]
I advice you to begin with this doc https://docs.blockscout.com/setup-and-run-blockscout/deployment/manual-deployment-guide

**Reply by kyonRay** [reply]
Hi @sl1depengwyn , 
Thank you very much for your reply, I have deployed blockscout locally using [docker-compose](https://docs.blockscout.com/setup-and-run-blockscout/deployment/docker-compose-deployment) way to deploy blockscout to connect to my local private blockchain, which using the geth chain type.

I found that I need to support some geth specific interfaces, such as JSON RPC interfaces like `txpool_content`.

My question is, what interfaces are necessary to adapt to blockscout? Could you please provide some doc about it?

Thank you for your time.

**Reply by GimluCom** [reply, marked as answer]
@kyonRay https://docs.blockscout.com/setup-and-run-blockscout/requirements/node-tracing-json-rpc-requirements
If you do not need pending transactions, you can disable it with `INDEXER_DISABLE_PENDING_TRANSACTIONS_FETCHER=true`

**Reply by kyonRay** [reply]
@GimluCom  Thank you very much for your answer, this doc is very useful!
